README.md

```diff
@@ -43,6 +43,13 @@ The implementation of `NADotAnnotation` places a red semi-transparent dot on the
 
 You can find a complete custom annotation example of multi-colored pins in [NAPinAnnotation.h](NAMapKit/NAPinAnnotation.h)/[.m](NAMapKit/NAPinAnnotation.m).
 
+####Circle popup menu
```

Comment:
I think this should just be added to above, just like we say "you can find a complete custom annotation example in " maybe say "another example of ... can be found in". Make sure to link the source files.